txn.go

```diff
@@ -343,13 +343,12 @@ func (txn *Txn) newPendingWritesIterator(reversed bool) *pendingWritesIterator {
 }
 
 func (txn *Txn) checkSize(e *Entry) error {
-	count := txn.count + 1
-	// Extra bytes for the version in key.
-	size := txn.size + int64(e.estimateSize(txn.db.opt.ValueThreshold)) + 10
-	if count >= txn.db.opt.maxBatchCount || size >= txn.db.opt.maxBatchSize {
+	if txn.count >= int(txn.db.opt.maxBatchCount) || txn.size >= int(txn.db.opt.maxBatchSize) {
 		return ErrTxnTooBig
 	}
-	txn.count, txn.size = count, size
+	txn.count++
```

Comment:
@CodeLingoBot capture this can be written in better way like this

txn.count = txn.count + 1`

txn.size += len(e.Key) + 8 + len(e.Value) + 2

Reply:
@CodeLingoBot ignore